index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable global-require */
+if (process.env.NODE_ENV === 'production') {
+  module.exports = require('./dist/reflex.min.js')
```

Comment:
Does flow understand this branching?

Reply:
not sure how to test it, I assumed it wouldn't matter for flow

Reply:
Just tested your changes on Iris and I don't think this works the way we want it to.